examples/parachain-example/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+//! # Tutorial: Subxt setup to interact with custom chains
```

Comment:
I think that both this and the WASM example should have a README.md so that it's immediately obvious where to look to find instructions on running them, so I'd move these docs into a README file instead I reckon.

Having great Rust docs for this (and the other example) crate isn't super important anyway since we don't publish them or expect anybody to consume the "apis" of them or whatever :)